### ISSUE TITLE
Idea: Add Wildlife to the Beaches, Jungles, Caves, and Ocean

### ISSUE THREAD

**Issue by tommy61157** [PiratesAhoy/new-horizons#85]
A cool little aesthetic thing could be to add some creatures for ambiance to the ocean other than sharks (Ex. Fish, Dolphins, Sea Turtles, maybe even a whale?) Add some creatures to the beaches (Ex. Crabs, Jellyfish and Fish near the shore) and Add some creatures to the Jungle maybe (Parrots/Exotic Birds, Snakes, iinsects, rodents?) and Add some creatures to caves and mines (Bats, rodents, insects, etc.)

Just would be cool to see them and maybe even have them make some noises. Just think it might be neat if certain environments like this had these sorts of things. Heck, in the ports, maybe you could even put in some fish near the port there for realism!

**Comment by Hammie** [comment]
Additional sea creatures should not be that hard.

But: we would need models and animations for all of those animals.

**Comment by tommy61157** [comment]
Yeah, longer term idea, but I think it would help make this feel like a truly distinct build and really add to the game.
